ambry-named-mysql/src/main/resources/NamedBlobsSchema.ddl

```diff
@@ -1,26 +1,3 @@
-CREATE TABLE IF NOT EXISTS named_blobs (
-    account_id int NOT NULL,
-    container_id int NOT NULL,
-    blob_name varchar(191) NOT NULL,
-    blob_id varbinary(50) NOT NULL,
-    deleted_ts datetime(6) DEFAULT NULL,
-    expires_ts datetime(6) DEFAULT NULL,
-    PRIMARY KEY (account_id, container_id, blob_name)
-)
-ENGINE=InnoDB
-DEFAULT CHARSET=utf8mb4
-COLLATE=utf8mb4_bin
-COMMENT='Holds mappings between blob names and blob IDs';
-
-/* Soft Delete Index */
-CREATE INDEX named_blobs_dt ON named_blobs(deleted_ts);
-
-/* Expired Index */
-CREATE INDEX named_blobs_et ON named_blobs(expires_ts);
-
-/* Reverse Lookup Index */
-CREATE INDEX named_blobs_id ON named_blobs(blob_id);
-
 /* New Table */
 CREATE TABLE IF NOT EXISTS named_blobs_v2 (
```

Comment:
QQ: Just want to make sure once you deprecate the named_blobs DB, you will rename the name_blobs_v2 to named_blobs right? I'm asking this is because auto purge feature will only apply to the named_blobs table.

Reply:
Will check with mysql team, i think we can either rename to named_blobs table or update auto-purge to refer to 'named_blob_v2' table?